### PR TITLE
add podman to `ch-convert`

### LINKED ISSUE #1360: `ch-convert`: teach it Podman

### PR DESCRIPTION
**Pull request by lucaudill**
closes #1360

### CODE REVIEW

**Review by reidpr — CHANGES_REQUESTED** [review]
Great work! Some comments in-line. Commit which hopefully works around the Podman problem pushed.

File: bin/ch-convert
Comment (lucaudill) [reply bin/ch-convert:309]: I opted to use `cv_docker_podman` and `cv_podman_docker` directly. It introduces some redundant code, but I thought it was less confusing/more readable.

File: bin/ch-convert
Comment (lucaudill) [reply bin/ch-convert:340]: `tr dp DP` doesn't work for podman because it has both a "p" and a "d", e.g.
```
$ echo podman | tr dp DP
PoDman
```
`case` was the best alternative I could think of

File: test/run/ch-convert.bats
Comment (reidpr) [inline test/run/ch-convert.bats:120]: This overreaches a bit (excludes all of `/run`, when we just want `/run` itself and `/run/.containerenv`, but that's probably fine.

**Review by reidpr — APPROVED** [review]
LGTM; nice work. I’m waiting on some other stuff so I want to wait until tomorrow to merge.

**Pull request merged**
